query/src/main/scala/filodb/query/exec/MultiSchemaPartitionsExec.scala

```diff
@@ -56,13 +57,22 @@ final case class MultiSchemaPartitionsExec(queryContext: QueryContext,
     (lookupRes, Some(columnName))
   }
 
+  def isMaxMinEnabledForWorkspace(ws: Option[String]) : Boolean = {
```

Comment:
Why does this accept an `Option`?

Reply:
```
  // scalastyle:off method.length
  private def finalizePlan(source: ChunkSource,
                           querySession: QuerySession): SelectRawPartitionsExec = {
    val partMethod = FilteredPartitionScan(ShardSplit(shard), filters)
    Kamon.currentSpan().mark("filtered-partition-scan")
    var lookupRes = source.lookupPartitions(dataset, partMethod, chunkMethod, querySession)
    val metricName = filters.find(_.column == metricColumn).map(_.filter.valuesStrings.head.toString)
    val ws = filters.find(_.column == TsCardinalities.LABEL_WORKSPACE).map(_.filter.valuesStrings.head.toString)
    var newColName = colName
```

This is because, when we find the string from filters, the search by default gives us an Option[String]. Even metricName is Option[String].

Reply:
Understood 👍